### PR TITLE
Answered!

### PR DESCRIPTION
**Pull request by rowlandhill**
3 of 5

### CODE REVIEW

**Comment by rowlandhill** [comment]
Ugh I messed up my last answer.  Oh well

File: lib/diagnostic.rb [hunk 1]
```diff
@@ -8,21 +8,21 @@
 # In a Ruby string, write the command you would use to run a script at
 # `lib/example.rb`
 
-Diagnostic.script = 'your response here'
+Diagnostic.script = 'require 'lib/example.rb''
```
Comment (scottyscripts) [inline lib/diagnostic.rb:12]: In ruby you can run scripts in ruby with the `ruby` keyword
`ruby 'lib/example.rb'`

File: lib/diagnostic.rb [hunk 2]
```diff
@@ -8,21 +8,21 @@
...
-Diagnostic.repl = 'your response here'
+Diagnostic.repl = pry
 
 # Naming conventions in Ruby
 #
 # In a Ruby string, write what you would write to properly name a variable in
 # Ruby that read in English as "star wars episode viii".
 
-Diagnostic.variable = 'your response here'
+Diagnostic.variable = 'SW8 = "star wars episode viii"'
```
Comment (scottyscripts) [inline lib/diagnostic.rb:26]: here we were looking for `star_wars_episode_viii`
remember that we use snake case to declare variables in Ruby

File: lib/diagnostic.rb [hunk 3]
```diff
@@ -38,24 +38,24 @@
 # What is the type of object in Ruby for decimal numbers? Write your answer in
 # a Ruby string.
 
-Diagnostic.decimal = 'your response here'
+Diagnostic.decimal = 'float'
```
Comment (scottyscripts) [inline lib/diagnostic.rb:42]: Great! Tip: Object types are usually capitalized.

File: lib/diagnostic.rb [hunk 4]
```diff
@@ -38,24 +38,24 @@
...
 
 # What is the type of object in Ruby for integer numbers? Write your answer in
 # a Ruby string.
 
-Diagnostic.integer = 'your response here'
+Diagnostic.integer = 'numbers are used as integer objects in Ruby'
```
Comment (scottyscripts) [inline lib/diagnostic.rb:47]: `Fixnum` is the Ruby object type for integer numbers

File: lib/diagnostic.rb [hunk 5]
```diff
@@ -108,10 +108,12 @@
 # Use shorthand to make this one line long.
 # Write your response as a Ruby string.
 
-Diagnostic.person_hash = 'your answer here'
+Diagnostic.person_hash = 'person = Hash.new
+person[:favorite_number] = [7]
+person[:first_name] = ["rowland"]'
```
Comment (scottyscripts) [inline lib/diagnostic.rb:114]: To make this one line long you could do 
`'person = {first_name: "Rowland", favorite_number: 7}'`

File: lib/diagnostic.rb [hunk 6]
```diff
@@ -108,10 +108,12 @@
...
 
 # Set a default return value to your `person` hash that concatenates the value
 # of person[:first_name] and a message about them.
 # Write your response as a Ruby string.
 
-Diagnostic.hash_default = 'your answer here'
+Diagnostic.hash_default = 'person.default = "#rowland.${first_name} is trying to learn Ruby"'
```
Comment (scottyscripts) [inline lib/diagnostic.rb:119]: Close! `person.default = "#{person[:first_name]} is great"`

**Comment by scottyscripts** [comment]
9/15 Nice!

**Pull request closed without merging**